README.md

```diff
@@ -414,24 +414,9 @@ CPMAddPackage(
 
 ### [Boost](https://github.com/boostorg/boost)
 
-Boost is a large project and will take a while to download. Using
-`CPM_SOURCE_CACHE` is strongly recommended. Cloning moves much more
-data than a source archive, so this sample will use a compressed
-source archive (tar.xz) release from Boost's github page.
+Boost has incompatible targets: for boost installed via b2 (or via CMake upto version boost_1.84.0 or via CMake scripts from `boost/tools/boost_install`) there's no targets for header-only Boost's libraries. Starting from boost_1.85.0 there's b2 version **and CMake version** of *install* targets, and Boost CMake version install CMake target even for header-only libraries, which allows installing and using only necessary boost libraries.
```

Comment:
How is this related to CPM? And why worry about installation?

Reply:
Because assume there's no boost on system, like any other library. What should cmake install when you asked to install your project that uses a boost library, when there's no boost on the specific system? Right, it installs the specific boost libraries too. That's why thinking about installation targets in this context is important.

Reply:
Hmmm... In my use cases, I compile boost as a static library so there's no boost installation. In the case of a library,  this gets more complicated. But I think the fix to patch Boost instead of add another dependency.

Reply:
> Hmmm... In my use cases, I compile boost as a static library so there's no boost installation. In the case of a library, this gets more complicated. But I think the fix to patch Boost instead of add another dependency.

I support your idea. I'll try realize it.

Reply:
Actually the script allows more than patching, but also easy linking, easy installing and so on, no matter is boost installed on the system or not, which is great.